enginetest/queries.go

```diff
@@ -8418,6 +8430,14 @@ var errorQueries = []QueryErrorTest{
 		Query:       `CREATE FULLTEXT INDEX idx ON opening_lines(opening_line)`,
 		ExpectedErr: sql.ErrUnsupportedFeature,
 	},
+	{
+		Query:       `SELECT * FROM datetime_table where date_col >= 'not a valid date'`,
+		ExpectedErr: sql.ErrConvertingToTime,
```

Comment:
Make this error string match what mysql gives:

`Incorrect %s value: '%s'` (name the type for the first param)